packages/zowe-explorer/README.md

```diff
@@ -23,25 +23,39 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.14](#whats-new-in-zowe-explorer-1.14)
+- [What's new in Zowe Explorer 1.15](#whats-new-in-zowe-explorer-1.15)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.14
+## What's new in Zowe Explorer 1.15
```

Comment:
Can we change this to 1.15.0

Reply:
We never had the patch version digit in Readme but I can add it :)